common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -340,9 +340,10 @@ def get_html(self, view=STUDENT_VIEW):
 
             if self.edx_video_id and edxval_api and video_status != u'external':
                 for index, source_url in enumerate(sources):
-                    new_url = rewrite_video_url(cdn_url, source_url)
-                    if new_url:
-                        sources[index] = new_url
+                    if any(cdn_prefix in source_url for cdn_prefix in ['edx-video.net', 'cloudfront.net']):
```

Comment:
This will not work. only CloudFront URLs, which are as per the edX pipeline signature, should be re-written. The course teams can have their content hosted on CloudFront different from edX signature. IT will also get re-written in this flow.

Reply:
@awaisdar001 we don't have the CloudFront CDN prefix defined. What would be the best approach to handle this?